src/sentry/utils/appleconnect/appstore_connect.py

```diff
@@ -0,0 +1,229 @@
+import logging
+import time
+from collections import namedtuple
+from typing import Any, Generator, List, Mapping, Optional
+
+import jwt
+from requests import Session
+
+from sentry.utils import safe
+
+logger = logging.getLogger(__name__)
+
+AppConnectCredentials = namedtuple("AppConnectCredentials", ["key_id", "key", "issuer_id"])
+
+
+def _get_authorization_header(credentials=AppConnectCredentials, expiry_sec=None) -> str:
+    """
+    Creates a JWT (javascript web token) for use with app store connect API
+
+    All requests to app store connect require an "Authorization" header build as below.
+
+    Note: Setting a very large expiry for the token will cause the authorization to fail,
+    the default
+
+    :return: the Bearer auth token to be added as the  "Authorization" header
+    """
+    if expiry_sec is None:
+        expiry_sec = 60 * 60  # default one hour
```

Comment:
I'd just do this in the parameter list as the default.  this isn't a type which can be modified.  But it doesn't really matter either way.